Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -132,11 +125,25 @@ void IPC_HLE_UpdateCallback(Core::System& system, u64 userdata, s64 cyclesLate)
   }
 }
 
+void GPUSleepCallback(Core::System& system, u64 userdata, s64 cyclesLate)
+{
+  auto& core_timing = system.GetCoreTiming();
+  system.GetFifo().GpuMaySleep();
+  core_timing.ScheduleEvent(VideoInterface::GetTicksPerField() / 8 - cyclesLate, et_GPU_sleeper);
```

Comment:
Is it possible for `cyclesLate` to ever be 8?  If so, this will be a divide by zero error.  Also, what is the significance of 8?

Reply:
I don't really know why, the old GPU sync used to happen once every millisecond. I just tried to have it go at a multiple of the VI events. I actually have no idea why this should be 8.